indexing-service/src/test/java/org/apache/druid/indexing/compact/CompactionSupervisorSpecTest.java

```diff
@@ -82,20 +82,32 @@ public void testSerdeOfSuspendedSpec()
   }
 
   @Test
-  public void testInvalidSpecThrowsException()
+  public void testSupervisorWithInvalidSpecThrowsExceptionForStatus()
```

Comment:
This test might have to change a little as we shouldn't throw an exception while getting state/status even if the spec is invalid.

Reply:
Updated.